setup.py

```diff
@@ -299,6 +299,9 @@ def write_version(filename: str = os.path.join(*["airflow", "git_version"])):
 postgres = [
     'psycopg2-binary>=2.7.4',
 ]
+presto = [
+    'presto-python-client>=0.7.0'
```

Comment:
This is probably a bit too loose. We should at least add 
```suggestion
    'presto-python-client>=0.7.0,<1'
```

And depending on the version history of this project `<0.8` might even make sense.

Reply:
Make sense.